torch/csrc/jit/passes/onnx/eval_peephole.cpp

```diff
@@ -47,14 +47,17 @@ static void fuseConvBatchNorm(Block* b, ValueToParamPairMap& valsToParamsMap) {
       fuseConvBatchNorm(child_block, valsToParamsMap);
     }
     if (it->kind() == onnx::Conv) {
-      if (it->output()->uses().size() != 1) {
+      auto oldConv = *it;
+      auto bnNode = oldConv->outputs().at(0)->uses()[0].user;
```

Comment:
Does this change mean that we expect the Conv op will have more than 1 output, not like before anymore?
If so, how can we assure that the first output is BatchNorm? Should we go through all of existing outputs to find out if BatchNorm op exists?

Reply:
No, I don't expect Conv node to have more than one outputs.
The use of "at(i)" would throw a runtime error instead of a seg-fault in case of an issue when we mess up the nodes.